src/CMakeLists.txt

```diff
@@ -661,6 +661,20 @@ target_link_libraries(MIOpen PRIVATE ${CMAKE_THREAD_LIBS_INIT} ${BZIP2_LIBRARIES
 generate_export_header(MIOpen
     EXPORT_FILE_NAME ${PROJECT_BINARY_DIR}/include/miopen/export.h
 )
+
+if(MIOPEN_ENABLE_AI_HEUR)
+    find_path(FDEEP_INCLUDE_DIR "fdeep/fdeep.hpp")
+    find_path(EIGEN_INCLUDE_DIR "eigen3/Eigen/Core")
+    target_include_directories(MIOpen SYSTEM PRIVATE $<BUILD_INTERFACE:${FDEEP_INCLUDE_DIR}>)
+    target_include_directories(MIOpen SYSTEM PRIVATE $<BUILD_INTERFACE:${EIGEN_INCLUDE_DIR}/eigen3>)
+    file(GLOB MODEL_FILES kernels/*.model)
+    install(FILES ${MODEL_FILES} DESTINATION ${DATA_INSTALL_DIR}/db)
```

Comment:
In DEV builds, the library should take models from the src/kernels. Is it supported?

Reply:
Like that. Please look how system-find-db files are handled, this should be the same, including embedding support (but the latter can be done in later PR).

Reply:
Added the check for Dev builds.

Reply:
This should work. Resolved.